content/sensu-go/5.12/reference/assets.md

```diff
@@ -235,26 +301,27 @@ spec:
 
 {{< /language-toggle >}}
 
-### Asset definition
+### Asset definition (single build)
```

Comment:
We're deprecating this single build style of assets. The recommended method is to add a single build to the `builds` array.